botocore/auth.py

```diff
@@ -815,7 +815,8 @@ def get_signature(
         self, method, split, headers, expires=None, auth_path=None
     ):
         if self.credentials.token:
-            del headers['x-amz-security-token']
+            if 'X-Amz-Security-Token' in headers:
+                del headers['X-Amz-Security-Token']
```

Comment:
or
```suggestion
            headers.pop('x-amz-security-token', None)
```

Reply:
Nope: https://github.com/boto/botocore/blob/97ba59024bdc313df6d18865f9ef233d24d60f95/botocore/auth.py#L841

Welcome to botocore :stuck_out_tongue:

Reply:
???? nvm then.. 😮